OpenRA.Game/Graphics/Sprite.cs

```diff
@@ -17,7 +17,6 @@ public class Sprite
 		public readonly Rectangle Bounds;
 		public readonly Sheet Sheet;
 		public readonly BlendMode BlendMode;
-		public readonly float Alpha;
```

Comment:
It no longer works after removing the `Translucency` blend mode, which was one of the main goals of this pr.  It is also completely superseded by the palette configuration.